packages/mobile/src/store/offline-downloads/sagas/downloadQueueSagas/downloadTrackWorker.ts

```diff
@@ -99,11 +98,5 @@ function* downloadTrackAsync(
     return OfflineDownloadStatus.ERROR
   }
 
-  const verified = yield* call(verifyTrack, trackId.toString(), true)
```

Comment:
is there a reason we don't want to do this anymore?

Reply:
yeah i thought that there was no way our methods above adding content to the filesystem would run without throwing if there was an issue with validity? basically thinking verifyTrack makes the most sense on the rehydrate phase